pyproject.toml

```diff
@@ -17,7 +17,7 @@ skip_glob = '^((?!py$).)*$'    # this makes sort all Python files
 
 [tool.poetry.dependencies]
 python = "^3.7"
-cognite-sdk-experimental = "^0.45.0"
+cognite-sdk-experimental = "^0"
```

Comment:
wut

Reply:
just in case it gets all the shit when you do `poetry update` major included

Reply:
I don't understand - we need at least a specific version, right?

Reply:
*minimum version

Reply:
![image](https://user-images.githubusercontent.com/40171187/109012849-c36eb200-76b2-11eb-9711-e9cb5b771233.png)